src/nvim/if_cscope.c

```diff
@@ -730,14 +730,14 @@ static int cs_create_connection(size_t i)
   int fd;
   SECURITY_ATTRIBUTES sa;
   PROCESS_INFORMATION pi;
-  BOOL pipe_stdin = FALSE, pipe_stdout = FALSE;  // NOLINT(readability/bool)
+  BOOL pipe_stdin = false, pipe_stdout = false;  // NOLINT(readability/bool)
   STARTUPINFO si;
   HANDLE stdin_rd, stdout_rd;
   HANDLE stdout_wr, stdin_wr;
   BOOL created;
 
   sa.nLength = sizeof(SECURITY_ATTRIBUTES);
-  sa.bInheritHandle = TRUE;
+  sa.bInheritHandle = true;
```

Comment:
Not very sure about this. Better leave this untouched.

Reply:
Does that mean we need to add the define back in?

Reply:
No. Try leaving this untouched but still removing the defines and see if code compiles.